tests/unit/uwb/protocols/fira/TestUwbFiraUwbConfigurationBuilder.cxx

```diff
@@ -15,7 +15,7 @@ constexpr uint32_t FiraVersionMacValue{ 0x1234ABCD };
 constexpr uint32_t FiraVersionPhyValue{ 0xDCBA4321 };
 constexpr Channel ChannelValue{ uwb::protocol::fira::Channel::C10 };
 constexpr DeviceRole DeviceRoleValue{ DeviceRole::Initiator };
-constexpr RangingMethod RangingMethodValue{ RangingDirection::OneWay, MeasurementReportMode::Deferred };
+constexpr RangingMethod RangingMethodValue{ RangingDirection::SingleSidedTwoWay, MeasurementReportMode::Deferred };

```

Comment:
Just curious: why this change?

Reply:
I believe OneWay, Deferred is not a valid RangingMethod so I changed this test to use a valid RangingMethod.